addons/mail/static/src/core/common/attachment_upload_service.js

```diff
@@ -82,7 +82,7 @@ export class AttachmentUploadService {
                 const threadModel = upload.data.get("thread_model");
                 const thread = this.store.Thread.get({ model: threadModel, id: threadId });
                 const attachment = this.store.Attachment.insert({
-                    ...response,
+                    ...(response?.result ?? response), // FIXME: this should be only response. HOOT tests returns wrong data {result, error}
```

Comment:
still to fix?

Reply:
Yes, but it takes time to fix mock server to handle that correctly, I prefer to make this quick-and-dirty fix for now to merge ASAP and fix this part of mock server eventually